### PR TITLE
Adding neumann chain_spec; unverified if this is functional

### PR DESCRIPTION
**Pull request by irsal**

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -177,6 +177,65 @@ pub fn local_testnet_config() -> ChainSpec {
...
+					(
+						get_account_id_from_seed::<sr25519::Public>("Alice"),
+						get_collator_keys_from_seed("Alice"),
+					),
+					(
+						get_account_id_from_seed::<sr25519::Public>("Bob"),
+						get_collator_keys_from_seed("Bob"),
+					),
+				],
+				vec![
```
Comment (irsal) [inline node/src/chain_spec.rs:206]: remove default accounts

**Comment by irsal** [comment]
Seems fine to keep the .json https://raw.githubusercontent.com/AcalaNetwork/Acala/master/resources/karura-dist.json

**Comment by irsal** [comment]
Open questions / thoughts:

 - polkadot-launch/config.json might not be useful (I have not isolated and tested removing this...though i assume it's not since we're NOT using the `polkadot-launch` tool
 - "chainType": "Local" vs "Live" vs something else (What does staging look like?)
 - how does --chain=neumann-staging tie to `chainType`
 - when i set  "chainType": "Live", all of the seed accounts go away
 - chaintype: Local, all of the seed accounts come back, what config do we need to make sure that "Live" seed accounts are not Alice, etc.

**Comment by irsal** [comment]
Woof, we had the wrong rebase config. We need to default to squash.

File: README.md [hunk 2]
```diff
@@ -121,6 +121,34 @@ cargo +nightly build --release
 
 ![image](https://user-images.githubusercontent.com/2915325/99548884-1be13580-2987-11eb-9a8b-20be658d34f9.png)
 
+
```
Comment (irsal) [inline README.md:124]: This README section is your friend. OnFinality docs are buggy/awful.

**Review by chrisli30 — APPROVED** [review]

**Review by chrisli30 — CHANGES_REQUESTED** [review]
This PR is not done yet. Need to rebase.

**Comment by imstar15** [comment]
I rebase this branch.

**Review by chrisli30 — APPROVED** [review]
Looks good to me, but let’s get more feedback from others.

**Comment by chrisli30** [comment]
> Looks good to me, but let’s get more feedback from others.

We can also check this in, deploy with the Docker image, and give it a try.

**Pull request merged**